hy/compiler.py

```diff
@@ -1614,33 +1641,39 @@ def compile_class_expression(self, expr, root, name, rest):
     def _rewire_init(self, expr):
         "Given a (setv …) form, append None to definitions of __init__."
```

Comment:
Pedantic nitpick on docstring: This function handles more than just `setv` forms now

Reply:
Fixed